lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -469,7 +484,7 @@ pub const MessagingDevicePath = union(Subtype) {
         device_id: u64,
     };
 
-    pub const UartDevicePath = packed struct {
+    pub const UartDevicePath = extern struct {
```

Comment:
There is a similar alignment problem here, with 2 bytes of padding between `reserved` and `baud_rate`.